cmd/kops/rollingupdatecluster.go

```diff
@@ -177,6 +181,7 @@ func NewCmdRollingUpdateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().BoolVar(&options.CloudOnly, "cloudonly", options.CloudOnly, "Perform rolling update without confirming progress with k8s")
 
 	cmd.Flags().DurationVar(&options.ValidationTimeout, "validation-timeout", options.ValidationTimeout, "Maximum time to wait for a cluster to validate")
+	cmd.Flags().Int32Var(&options.ValidateTimes, "validate-times", options.ValidateTimes, "Amount of times that a cluster needs to be validated after single node update")
```

Comment:
For validate we have `--count`. Would be nice to keep it consistent.
```suggestion
	cmd.Flags().Int32Var(&options.ValidateTimes, "validate-count", options.ValidateTimes, "Amount of times that a cluster needs to be validated after single node update")
```